document_page/models/document_page_history.py

```diff
@@ -0,0 +1,72 @@
+"""DocumentPageHistory model."""
```

Comment:
It's not advised to the docstrings on python files. Add to Model classes instead.